Wheres-Waldo-Frontend/src/tests/HomePage.test.jsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * @jest-environment jsdom
+ */
+import { render, fireEvent, screen } from "@testing-library/react";
+import { expect, test } from "vitest";
+import HomePage from "../components/HomePage";
+import { MemoryRouter } from "react-router-dom";
+
+test("HomePage tests", async () => {
+
+    //renders homepage component
+    render(<MemoryRouter><HomePage /></MemoryRouter>);
+
+    //title and gamerules display on the screen
+    expect(screen.getByRole('title'))
+    expect(screen.getByRole('gameRules'))
```

Comment:
Generally it's better to test by text instead of role if possible, the idea is to test in the same way a user would test the page.

This is the article people usually link to: https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-querying-by-text